cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/evaluation/ValueEvaluator.kt

```diff
@@ -426,6 +426,7 @@ open class ValueEvaluator(
         var condition = expr.condition
 
         // Assume that condition is a binary operator
+        // Todo assume
```

Comment:
rather than writing `// TODO assume´ I would appreciate if we would actually already use `assume` here and set the message to "TODO", but this way we can at least search for all assume functions